openfl-tutorials/interactive_api/HPU/PyTorch_Kvasir_UNet/envoy/envoy_config.yaml

```diff
@@ -0,0 +1,14 @@
+params:
+  cuda_devices: [0,2]
+  
+optional_plugin_components:
+ cuda_device_monitor:
```

Comment:
In general, please remove all cuda-related explicit parameters from the PR.

Reply:
I have removed cuda-related explicit parameters from where ever necessary. This example initially had 2 envoy cofig files , one with gpu params and one without gpu. To avoid confusion I have only retained non-gpu config